### PR TITLE
Resolve todos

### PR DESCRIPTION
**Pull request by matheusd**
This fixes and removes some outstanding/superseded TODOs left during initial conversion.

### CODE REVIEW

File: chainregistry.go [hunk 1]
```diff
@@ -30,15 +30,18 @@ import (
 
 const (
 	// TODO(decred) verify these amounts
```
Comment (fguisso) [inline chainregistry.go]: this todo has done?

File: input/signdescriptor.go [hunk 2]
```diff
@@ -71,9 +71,10 @@ type SignDescriptor struct {
 	// generating the final sighash, and signature.
 	HashType txscript.SigHashType
 
-	// TODO(decred): Fix...
 	// SigHashes is the pre-computed sighash midstate to be used when
 	// generating the final sighash for signing.
+	//
+	// Note: Decred doesn't currently use a sighash cache.
```
Comment (fguisso) [inline input/signdescriptor.go]: Why not remove?

File: chainregistry.go (hunk 1 above)
Comment (matheusd) [reply chainregistry.go]: We haven't done any community-wide discussion regarding the minimum htlc amounts, timelocks, fees and feerates, etc so I'm leaving this as is for the moment until we get more usage out of our LN and can make a better informed decision on changing or leaving these as is.

File: input/signdescriptor.go (hunk 2 above)
Comment (matheusd) [reply input/signdescriptor.go]: Removed.

**Pull request merged**